AutorestSwift.xcodeproj/xcshareddata/xcschemes/AutorestSwift.xcscheme

```diff
@@ -60,6 +60,12 @@
             ReferencedContainer = "container:AutorestSwift.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "--standalone"
+            isEnabled = "YES">
```

Comment:
how will isEnabled being used?

Reply:
There's a checkbox in the UI that you can use to toggle it off.